app/gateway-oss/2.5.x/db-less-and-declarative-config.md

```diff
@@ -204,30 +204,44 @@ parse successful
 
 ## Loading The Declarative Configuration File
 
-There are two ways to load a declarative configuration into Kong: via
-`kong.conf` and via the Admin API.
+There are two ways to load a declarative configuration file into Kong: using
+`kong.conf` or the Admin API.
 
-To load a declarative configuration at Kong start-up, use the
+To load a declarative configuration file at Kong start-up, use the
 `declarative_config` directive in `kong.conf` (or, as usual to all `kong.conf`
 entries, the equivalent `KONG_DECLARATIVE_CONFIG` environment variable).
 
 ```
-$ export KONG_DATABASE=off
-$ export KONG_DECLARATIVE_CONFIG=kong.yml
-$ kong start -c kong.conf
+export KONG_DATABASE=off \
+export KONG_DECLARATIVE_CONFIG=kong.yml \
+kong start -c kong.conf
 ```
 
-Alternatively, you can load a declarative configuration into a running
-Kong node via its Admin API, using the `/config` endpoint. The
+You can also load a declarative configuration file into a running
+Kong node by its Admin API, using the `/config` endpoint. The
 following example loads `kong.yml` using HTTPie:
 
 ```
 $ http :8001/config config=@kong.yml
 ```
 
-The `/config` endpoint replaces the entire set of entities in memory
+{:.important}
+> The `/config` endpoint replaces the entire set of entities in memory
 with the ones specified in the given file.
 
+Alternatively, you can start Kong in DB-less mode with a minimum
+declarative configuration in a string using the `KONG_DECLARATIVE_CONFIG_STRING`
+environment variable. 
```

Comment:
This feels like it's heading into run-on territory (hasn't quite landed yet, but here's my effort). Making it a bit more casual too, in the spirit of conversational tone:

```suggestion
Or to start in DB-less mode, provide a minimum declarative configuration with a string you pass with the `KONG_DECLARATIVE_CONFIG_STRING` environment variable. 
```